src/bp-core/classes/class-bp-admin-tab.php

```diff
@@ -109,67 +109,84 @@ public function register_admin_script() {
 
 			$cover_dimensions = bb_attachments_get_default_custom_cover_image_dimensions();
 
-			wp_localize_script(
-				'bp-admin',
-				'BP_ADMIN',
-				array(
-					'ajax_url'            => admin_url( 'admin-ajax.php' ),
-					'select_document'     => esc_js( __( 'Please upload a file to check the MIME Type.', 'buddyboss' ) ),
-					'tools'               => array(
-						'default_data'  => array(
-							'submit_button_message' => esc_js( __( 'Are you sure you want to import data? This action is going to alter your database. If this is a live website you may want to create a backup of your database first.', 'buddyboss' ) ),
-							'clear_button_message'  => esc_js( __( 'Are you sure you want to delete all Default Data content? Content that was created by you and others, and not by this default data installer, will not be deleted.', 'buddyboss' ) ),
-						),
-						'repair_forums' => array(
-							'validate_site_id_message' => esc_html__( 'Select site to repair the forums', 'buddyboss' ),
-						),
+			$localize_arg = array(
+				'ajax_url'            => admin_url( 'admin-ajax.php' ),
+				'select_document'     => esc_js( __( 'Please upload a file to check the MIME Type.', 'buddyboss' ) ),
+				'tools'               => array(
+					'default_data'  => array(
+						'submit_button_message' => esc_js( __( 'Are you sure you want to import data? This action is going to alter your database. If this is a live website you may want to create a backup of your database first.', 'buddyboss' ) ),
```

Comment:
```suggestion
						'submit_button_message' => esc_js( esc_html__( 'Are you sure you want to import data? This action is going to alter your database. If this is a live website you may want to create a backup of your database first.', 'buddyboss' ) ),
```

Reply:
JI - This was not related to this PR but I have done this.